src/com/lftechnology/java/training/alina/basic/packagejarexecute/NumberPrint.java

```diff
@@ -0,0 +1,59 @@
+
+package com.lftechnology.java.training.alina.basic.packagejarexecute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <p>NumberPrint Class displays numbers from 1-10.</p>
+ * 
+ * <p>Steps : </p>
+ * <ol type="a">
+ * <li>Create a Java application PrintNumber which prints number from 1-10.</li>
+ * <li>Create Executable Jar</li>
+ * <li>Run java application from command line.</li>
+ * <li>Print numbers</li>
+ * <li>Create JavaDoc of Application</li>
+ * </ol>
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class NumberPrint {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)